### PR TITLE
Update polyfill

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

**Comment by nickvergessen** [comment]
The diff looks confusing, because the polyfill stuff actually didn't change, just the dependency of those?

**Comment by rullzer** [comment]
Yes it seem so. It is just a version bump with some dep upgrades as I understand it.

**Comment by rullzer** [comment]
Just want to get 3rdparty deps a bit more in shape again ;)

File: symfony/polyfill-php70/composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "paragonie/random_compat": "~1.0"
+        "paragonie/random_compat": "~1.0|~2.0"
```
Comment (LukasReschke) [inline symfony/polyfill-php70/composer.json]: I wouldn't bump to 2.0. This breaks compatibility: https://github.com/paragonie/random_compat/blob/master/CHANGELOG.md#version-200---2016-03-18

**Comment by nickvergessen** [comment]
:+1:

File: symfony/polyfill-php70/composer.json (hunk 1 above)
Comment (nickvergessen) [inline symfony/polyfill-php70/composer.json]: That is the dep of our dep, not a file we touch

Comment (LukasReschke) [inline symfony/polyfill-php70/composer.json]: Yeah. But using 2.0 here will break stuff. We can't take that. Sorry 👎

Comment (LukasReschke) [inline symfony/polyfill-php70/composer.json]: Also see the diff above, the 1.0 version has been replaced with 2.0. We should keep 1.0.x here for now.

Comment (LukasReschke) [inline symfony/polyfill-php70/composer.json]: Basically with 2.0 instances where /dev/urandom is not readable will fail pretty hard ;)

**Comment by nickvergessen** [comment]
Since we drop the 5.4 and 5.5 support we can drop polyfill 5.5 snd 5.6?

**Comment by MorrisJobke** [comment]
> Since we drop the 5.4 and 5.5 support we can drop polyfill 5.5 snd 5.6?

See #11 ;)

**Pull request closed without merging**